addons/hide-backpack/userscript.js

```diff
@@ -0,0 +1,80 @@
+export default async function ({ addon }) {
+  let originalBackpack = await addon.tab.waitForElement("[class^=backpack_backpack-header_]", {
+    markAsSeen: true,
+  });
+
+  changeBackpackVisibility(addon);
+
+  // Event listeners that add dynamic enable/disable + setting change
+  addon.settings.addEventListener("change", () => changeBackpackVisibility(addon));
+  addon.self.addEventListener("reenabled", () => changeBackpackVisibility(addon));
+  addon.self.addEventListener("disabled", () => {
+    moveResizeButtons(0);
+    originalBackpack.style.display = "block";
+    window.dispatchEvent(new Event("resize"));
+    document.querySelector(".sa-backpack-button").style.display = "none";
+  });
+
+  while (true) {
+    originalBackpack = await addon.tab.waitForElement("[class^=backpack_backpack-header_]", {
+      markAsSeen: true,
+    });
```

Comment:
Possible optimization (needs to be tested)
`reduxCondition: (state) => state.scratchGui.editorTab.activeTabIndex === 0 && !state.scratchGui.mode.isPlayerOnly,`

Reply:
It doesn't work because of `state.scratchGui.editorTab.activeTabIndex === 0` (normally you open project in first page, but if you click see inside it doesn't work), but we can just add `reduxCondition: (state) => !state.scratchGui.mode.isPlayerOnly,`